packages/components/src/dialog/src/Dialog.css

```diff
@@ -28,6 +28,7 @@
     align-items: stretch;
     grid-template-areas: "image image" "aside aside";
     grid-template-rows: min-content auto;
+    width: 28.75rem;
```

Comment:
This doesn't seem right, why hardcode a with on the dialog

Reply:
Just removed when I noticed.